src/pages/home/report/ReportActionCompose.js

```diff
@@ -57,7 +57,7 @@ const propTypes = {
     comment: PropTypes.string,
 
     /** The ID of the report actions will be created for */
-    reportID: PropTypes.number.isRequired,
+    reportID: PropTypes.string.isRequired,
```

Comment:
I guess we need to get clarification on that. I thought they were supposed to be strings everywhere so that when they are upgraded to 64bit integers, they would work out-of-the-box with no other changes.

Reply:
I think we should already use strings in App, since as Tim said we will be moving them to 64bit numeric strings later on.

Reply:
I am confused about whether reportID are all strings currently? Has that change happened already?

Reply:
# Conclusion from Slack
ReportIDs should be strings everywhere in JS.

I am creating a separate issue: https://github.com/Expensify/Expensify/issues/227425 which will clean this up from the source and make it a string everywhere consistently. For now, this PR is going to be a little bit janky with adding a few `toString()` casts where they are needed.